lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -169,6 +169,7 @@ export default TestModule.extend({
       Ember.run(function() {
         Ember.setProperties(context, hash);
       });
+      return hash;
```

Comment:
I agree that the Ember behavior changing between 1.x and 2.x is a gotcha, but I'd rather stick with what the framework does here (what if behavior changes again in the future?).  Also, the same behavior change exists for `this.set`, so we should be consistent between these two.

I think I would prefer to leave the return value undefined (current behavior) for Ember 1.x, and use Ember's return value for 2.x.

This might look something like this:

```
    context.set = function(key, value) {
      var ret = Ember.run(function() {
        return Ember.set(context, key, value);
      });

      if (hasEmberVersion(2,0)) {
        return ret;
      }
    };

    context.setProperties = function(hash) {
      var ret = Ember.run(function() {
        return Ember.setProperties(context, hash);
      });

      if (hasEmberVersion(2,0)) {
        return ret;
      }
    };
```

What do you think?